src/Nancy.Testing/Browser.cs

```diff
@@ -54,9 +52,10 @@ public Browser(INancyBootstrapper bootstrapper, Action<BrowserContext> defaults
         /// <param name="path">The path that is being requested.</param>
         /// <param name="browserContext">An closure for providing browser context for the request.</param>
         /// <returns>An <see cref="BrowserResponse"/> instance of the executed request.</returns>
+        [Obsolete("Use async overload instead.", false)]

```

Comment:
I'm not saying we set it to `true`, it just wasn't obvious what `false` meant...